### ISSUE TITLE
[MU4 Issue] Windows 11 Snap layouts not shown in MU4

### ISSUE THREAD

**Issue by GabeS573** [musescore/MuseScore#10965]
**Describe the bug**
When hovering over the Maximize button in the toolbar of MuseScore, the snap layouts are not displayed.

Also, if MuseScore is snapped next to another app, it often is too large to fit the available space, probably due to the minimum size of the app window. See screen recording below.

**To Reproduce**
1. Open MuseScore
2. Hover over the Maximize button (do not click on it)
3. The snap layouts do not appear.
4. Open another app (i.e. notepad) and snap it to the left of the screen using the snap layouts.
5. When shown which other apps you can snap to the right side of screen, click on MuseScore
6. Depending on your screen size, MuseScore may be too large to fit the space.

**Expected behavior**
Snap layouts should be available when hovering over the maximize button, just like they were in MuseScore 3. Also, when snapped, MuseScore should fit the available space like it did in MuseScore 3 (screenshot below).

**Screenshots**

Video:

https://user-images.githubusercontent.com/90187801/160885524-1e9b2bc5-fe99-4f6c-bbe7-26a80e090f46.mp4

MuseScore 3:

![Screenshot 2022-03-29 203455](https://user-images.githubusercontent.com/91428485/160746324-9d090e2e-0fae-419a-88d7-7c1a9ef86bd9.jpg)

MuseScore 4:

![Screenshot 2022-03-29 203430](https://user-images.githubusercontent.com/91428485/160746285-cfb4264c-34be-496d-85b5-1dca8f0dabf6.jpg)

**Platform information**
 - OS: Windows 11

**Additional context**
As noted above, snap layouts worked perfectly with MuseScore 3. For the issue of the window not being small enough to snap correctly next to other apps, this may be because MuseScore 3 had a very small minimum window size.

**Comment by GabeS573** [comment]
For reference, here are the minimum sizes of MuseScore 3 and MuseScore 4:

![Screenshot 2022-03-29 203911](https://user-images.githubusercontent.com/91428485/160746724-b263ac83-053e-4b4a-962d-4e92f676cb82.jpg)

**Comment by DmitryArefiev** [comment]
Thanks for logging!

(I converted the attached video and re-added to the description to be able to watch it inside web browser )

**Comment by Tantacrul** [comment]
@cbjeukendrup - can't this simply be fixed by lowering the minimum size of MS4? I seem to recall we discussed this on another issue.

**Comment by cbjeukendrup** [comment]
Yes, it's related to #11627. Lowering the minimum width should fix this issue, but that will cause all sorts of layout problems. Because if we set the minimum width of the window too low, the docking system is not able anymore to realize the specified minimum width of the toolbars and panels. The result is that it completely stops managing the layout and starts shouting error messages. So that prevents us from just lowering the minimum width.

**Comment by Tantacrul** [comment]
What is the lowest number we can set the minimum size to? Basically, every pixel counts.

**Comment by GabeS573** [comment]
This is partially a duplicate of #11083, so I'm closing it as not planned. I'll open the other half of this as a new issue.
